lib/observer/observer.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Observer do
+require Satellite.Constants
+  defstruct [:latitude, :longitude, :height]
+
+  def create_from(latitude_degrees, longitude_degrees, height_km) do
```

Comment:
But yes I think all the public API into the library should probably be in degrees.